Configurateur/front/lib/screens/landing-page/landing_page.dart

```diff
@@ -192,196 +46,138 @@ class LandingPageState extends State<LandingPage> {
-                          ),
-                        ),
+                    ],
+                  )),
+                ),
+                const SizedBox(height: 50),
+                Container(
+                  padding: const EdgeInsets.all(20.0),
+                  child: Center(
+                      child: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      Image.asset(
+                        'assets/containerrisu.png',
                       ),
-                    )
-                  ]),
-                ]),
-            Image.asset(
-              'assets/iphone.png',
+                      const SizedBox(width: 100),
+                      Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(
+                              'Concevez le conteneur de vos rêves,\nselon vos envies !',
+                              style: TextStyle(
+                                fontSize: 35,
+                                color: Provider.of<ThemeService>(context).isDark
+                                    ? darkTheme.secondaryHeaderColor
+                                    : lightTheme.secondaryHeaderColor,
+                                fontWeight: FontWeight.bold,
+                                shadows: const [
+                                  Shadow(
+                                    color: Color(0xff28666e),
+                                    offset: Offset(0.75, 0.75),
+                                    blurRadius: 1.5,
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(
+                                  top:
+                                      15), // Espacement inférieur pour le texte
+                              child: Text(
+                                'Grâce à notre configurateur innovant,\nvotre conteneur sera à la hauteur de vos attentes',
+                                style: TextStyle(
+                                  fontSize: 20,
+                                  color:
+                                      Provider.of<ThemeService>(context).isDark
+                                          ? darkTheme.secondaryHeaderColor
+                                          : lightTheme.secondaryHeaderColor,
+                                ),
+                              ),
+                            ),
+                          ]),
+                    ],
+                  )),
+                ),
+              ],
             ),
-          ],
-        )),
-      ),
-      bottomNavigationBar: const CustomBottomNavigationBar(),
+          ]),
```

Comment:
Maybe add , after the ] so that the file is more readable

Reply:
done